Unix/tests/http/test_http.cpp

```diff
@@ -43,7 +43,9 @@ NitsSetup(TestHttpSetup)
 {
     Sock_Start();
     PORT++;
+    IgnoreAuthCalls(1);
```

Comment:
I'm not sure what this does, but the rest is fine with me. I imagine this has to do with PAM stuff. Signed off on everything else!